chat/src/main/res/layout/conversation_fragment.xml

```diff
@@ -2,4 +2,13 @@
 <io.skygear.plugins.chat.ui.ConversationView
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    xmlns:android="http://schemas.android.com/apk/res/android" />
\ No newline at end of file
+    uikit:avatarNameField="username"
+    uikit:avatarImageField="avatar"
+    uikit:avatarShowSender="true"
+    uikit:avatarShowReceiver="true"
+    uikit:senderUserNameTextColor="@color/cornflower_blue_light_40"
+    uikit:avatarBackgroundColor="@color/blue_1"
+    uikit:avatarInitialTextColor="@color/dark_gray"
```

Comment:
We had the default in ConversationView constructor already. So I think we should not set this in `conversation_fragment.xml`. Otherwise we will have 2 default. Also for the default value, please use the color and setting that stick with the original design.